website/package.json

```diff
@@ -51,6 +52,7 @@
     "@crowdin/cli": "^3.5.2",
     "@docusaurus/module-type-aliases": "^3.0.0",
     "@docusaurus/tsconfig": "^3.0.0",
+    "@jest/schemas": "workspace:*",
```

Comment:
I'll add `"@jest/schemas-30x": "npm:@jest/schemas@^30" etc later for different versions